examples/firebase/index.js

```diff
@@ -0,0 +1,179 @@
+const functions = require("firebase-functions");
+const { PKPass } = require("passkit-generator");
+const admin = require("firebase-admin");
+var fs = require('file-system');
+var path = require('path');
+var axios = require('axios');
+var os = require('os');
+
+// Firebase init
+admin.initializeApp({
+    credential: admin.credential.cert(require("CERTIFICATE_PATH")),
+    storageBucket: "STORAGE_BUCKET_URL"
+});
+var storageRef = admin.storage().bucket()
+
+exports.pass = functions.https.onRequest((request, response) => {
+    PKPass.from({
+        // Get relevant pass model from model folder (see passkit-generator/examples/models/)
+        model: `./model/${request.body.passType}.pass`,
+        certificates: {
+            // Assigning certificates from certs folder (you will need to provide these yourself)
+            wwdr: fs.fs.readFileSync("./certs/wwdr.pem"),
```

Comment:
Wouldn't using `fs` on a cloud function make the cloud function crash? Maybe it is better to change paths here to `process.env` o something like this.

Reply:
It's not crashed from what I've seen? I've been generating passes using this function for a few months now, besides a few bits with the images not coming up on the pass (which I think is more to do with my frontend anyway), I haven't really had any issues with this. If you think there is a better way to do this though let me know, I'd be happy to change it

Reply:
I'm wondering how it could read the files if Firebase cannot use FileSystem API 🤔